c2rust-analyze/src/dataflow/type_check.rs

```diff
@@ -280,7 +280,7 @@ impl<'tcx> TypeChecker<'tcx, '_> {
                 match callee {
                     Callee::Trivial => {}
                     Callee::UnknownDef { .. } => {
-                        todo!("visit Callee::{callee:?}");
+                        eprintln!("visit Callee::{callee:?}");
```

Comment:
given that `eprintln!` is used everywhere else, could we make it even more obvious that this is unknown and unvisited?

Reply:
It should be clearer with the `log::error!` and `[ERROR @ ...]` printed now, I think.